.github/workflows/post-release.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         run: sudo npm install -g github-changes
 
       - name: Update changelog
-        run: github-changes -o tardis-sn -r tardis --only-pulls --use-commit-body -f CHANGELOG.md -k ${{ secrets.BOT_TOKEN }}
```

Comment:
Were you able to run the command locally?

Reply:
We'll need the token
https://stackoverflow.com/questions/33655700/github-api-fetch-issues-with-exceeds-rate-limit-prematurely